Soft Allocation Journey.md

```diff
@@ -0,0 +1,193 @@
+
+### Sample of `checkCartInventory`:
+
+### Method : `/POST`
+
+## Request
+
+```
+{
+  point: "\\<latlon\\>",
+  distance: "sameDayStoreSearchProximity",
+  shopifyShopId: "shopId",
+  items: "cartItems",
+  filters: ["\\<filter1\\>", "\\<filter2\\>"]
+}
+```
+
+## Response
+
+```
+{
+  "facilityId": "646",
+  "distance": 15,
+  "cors.request.origin": "https://originalpenguin-dev.myshopify.com",
+  "shopifyShopId": "10224533555",
+  "filters": [
+    "sameday_pref: true",
+    "docType: STORE",
+    "shopifyShop_id:10224533555"
+  ],
+  "point": "25.786909, -80.361253",
+  "cors.request.type": "actual",
+  "items": [{
+    "quantity": 1,
+    "variantId": "42088603123903",
+    "availableQty": 1,
+    "status": "available"
+  }],
+  "facility": {
+    "docType": "STORE",
+    "storeCode": "646",
+    "identifier": "646",
+    "storeType": "RETAIL_STORE",
+    "storeName": "Dolphin Mall",
+    "externalId": "646",
+    "primaryFacilityGroupId": "10224533555",
+    "primaryFacilityGroupName": "OP_Group",
+    "primaryShopifyShopId": "10224533555",
+    "10224533555_pref": "true",
+    "fac_grp_pref": "true",
+    "sameday_pref": "true",
+    "address1": "11401 NW 12th Street",
+    "city": "Miami",
+    "postalCode": "33172",
+    "country": "United States",
+    "countryCode": "US",
+    "state": "Florida",
+    "stateCode": "FL",
+    "latlon": "25.7875845,-80.3805674",
+    "shopifyShop_id": ["10224533555"],
+    "shopifyShops": [{
+      "domain": "originalpenguin-dev.myshopify.com",
+      "name": "originalpenguin-dev",
+      "myshopifyDomain": "originalpenguin-dev.myshopify.com",
+      "shopifyShopId": "10224533555",
+      "shopId": "OP_SHOP"
+    }],
+    "productStore_id": ["OP_STORE"],
+    "productStores": [{
+      "storeName": "Original Penguin",
+      "productStoreId": "OP_STORE"
+    }],
+    "monday_open": "10:00:00",
+    "monday_close": "21:00:00",
+    "tuesday_open": "10:00:00",
+    "tuesday_close": "21:00:00",
+    "wednesday_open": "10:00:00",
+    "wednesday_close": "21:00:00",
+    "thursday_open": "10:00:00",
+    "thursday_close": "21:00:00",
+    "friday_open": "10:00:00",
+    "friday_close": "21:00:00",
+    "saturday_open": "10:00:00",
+    "saturday_close": "21:00:00",
+    "sunday_open": "11:00:00",
+    "sunday_close": "20:00:00",
+    "updatedDatetime": "2023-06-15T13:41:09.708Z",
+    "docType-identifier": "STORE-646",
+    "_version_": 1768776238064730000,
+    "dist": 1.9352518
+  }
+}
+```
+
+## Step 3: Displaying Eligibility on the Cart Page 
+
+Upon receiving the response indicating stores eligible for same-day delivery, the application follows a systematic comparison process:
+
+1. Lead Time Comparison:
+   - The application compares the order preparation lead time with the difference between the customer's current time and the store's closing time.
```

Comment:
```suggestion
1. **Lead Time Comparison:** The application compares the order preparation lead time with the difference between the customer's current time and the store's closing time.
```

Also is the lead role coded into the app? I think the lead time is saved in the oms and sent with facility data. @Dhiraj1405 Please double check